### PR TITLE
Added solution for HW6.1

### PR DESCRIPTION
**Pull request by AlexanderSashchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/Pair.java [hunk 1]
```diff
@@ -1,5 +1,41 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, K> {
 
+    T first;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Pair.java]: add access modifier

File: src/main/java/core/basesyntax/Pair.java [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+    K second;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Pair.java]: same

File: src/main/java/core/basesyntax/Pair.java (hunk 1 above)
Comment (AlexanderSashchenko) [reply src/main/java/core/basesyntax/Pair.java]: Fixed

File: src/main/java/core/basesyntax/Pair.java (hunk 2 above)
Comment (AlexanderSashchenko) [reply src/main/java/core/basesyntax/Pair.java]: Done

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**